app/Console/ImportSavedSearchesCommand.php

```diff
@@ -86,7 +97,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             $output->writeln('Importing query with name: ' . $record['NAME']);
 
-            $this->getUDB3SavedSearchesRepository($input)->write(
+            $this->repositoryFactory->createForVersion($input->getOption(self::SAPI_VERSION))->write(
```

Comment:
Do we still use SAPI version 2 here? @bertramakers ?

Reply:
There is still a table for saved searches for sapi2, but it should be removed in the near future.
There's even a ticket to remove all dependencies on sapi2 from udb3 in the next up column: https://jira.uitdatabank.be/browse/III-3097
I've added a comment to also remove the code for sapi2 saved searches and role constraints from the backend.

So I think you could directly inject the sapi3 repository instead and remove the `sapi_version` option.

Thinking a bit further though, I'm not even sure if this command is still needed. Since most are only run manually by a select few people, we could check with @willaerk and @corneelwille which ones we actually still need and remove the obsolete ones.